.asf.yaml

```diff
@@ -0,0 +1,45 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# https://cwiki.apache.org/confluence/display/INFRA/.asf.yaml+features+for+git+repositories
+---
+github:
+  description: "Apache Superset is a Data Visualization and Exploration Platform"
+  homepage: https://superset.apache.org/
+  labels:
+    - superset
+    - apache
+    - apache-superset
+    - data-visualization
+    - data-viz
+    - analytics
+    - business-intelligence
+    - data-science
+    - data-engineering
+    - asf
+  features:
+    # Enable issues management
+    issues: true
+    # Enable projects for project management boards
+    projects: true
+    # Enable wiki for documentation
+    wiki: true
```

Comment:
Do we want this to be true? I don't think we've ever really used github wiki for documentation, instead using either the `CONTRIBUTING.md` file or `superset.apache.org`

Reply:
It came up recently as someone was asking about using Confluence, as people might want to craft design documents that aren't SIPs, or have pre-SIP documents.

My immediate reaction was "please no vintage-asf-hosted-Confluence!" let's turn on the Wiki instead. 

I agree that generally publish content should be reviewed, and that between things like `superset.apache.org`, `CONTRIBUTING.md`, `stackoverflow.com` and other resources we have, the use-cases for the wiki are pretty thin.

Could be a decent place for people to publish things and collaborate while things settle, prior to the act of publishing, or for things that are transient.

Reply:
I actually find modern Confluence very easy to use and much more powerful than GitHub wiki!